docs/admin-ports.md

```diff
@@ -38,6 +38,8 @@ The following is a list of ports used by internal DC/OS services, and their corr
  - 8181: dcos-exhibitor
  - 9990: dcos-cosmos
  - 15055: dcos-history-service
+ - 15101: dcos-marathon libprocess
+ - 15201: dcos-metronome libprocess
```

Comment:
Why not follow `8080: dcos-marathon` with 8081 for marathon libprocess?
Does Metronome have a TCP port for its own API, not listed here?